app/src/utilities/Formatters.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Convert money strings to locale formatted money string
+ * @param currency 3 digit currency string
+ * @param amount amount without decimal place
+ * @returns formatted string
+ */
+export const MoneyFormatter = (currency: string, amount: string) => {
+  const formatter = new Intl.NumberFormat(undefined, {
+    style: "currency",
+    currency: currency,
+  });
+  return formatter.format(parseInt(amount) / 100);
```

Comment:
Should this be /100? I think it varies by currency

Reply:
ah good point, I need to use a currency library

Reply:
I'm going to make this an issue today, and fix soon, rather than in this commit